libraries/cli/include/cli/devnet_config.hpp

```diff
@@ -15,8 +15,8 @@ constexpr std::string_view devnet_json = R"foo({
   "network_simulated_delay": 0,
   "network_transaction_interval": 100,
   "network_bandwidth": 40,
-  "network_ideal_peer_count": 5,
-  "network_max_peer_count": 15,
+  "network_ideal_peer_count": 15,
+  "network_max_peer_count": 30,
```

Comment:
But that will also change for consensus nodes. 
I think the fix should only change on RPC nodes, and not for consensus nodes

Reply:
it does not matter how many consensus node have it is devnet. Anyway we increased those values for rest of the networks - mainnet, testnet